test/e2e/node/kubelet.go

```diff
@@ -385,6 +386,102 @@ var _ = SIGDescribe("kubelet", func() {
 		}
 	})
 
+	ginkgo.Describe("Delete termination message log files when container is removed", func() {
```

Comment:
Can we have the same test for init container (maybe for sidecar as well). Also I am not sure if anything changes if `restartPolicy: Always` and container returns `0` and `1` semi-randomly. Is there any logic around termination log that depends on whether container exited sucessfully or failed?